napari/layers/points/_points_utils.py

```diff
@@ -45,17 +86,85 @@ def points_to_squares(points, sizes):
     """
     rect = np.concatenate(
         [
-            points + np.sqrt(2) / 2 * np.array([sizes, sizes]).T,
-            points + np.sqrt(2) / 2 * np.array([sizes, -sizes]).T,
-            points + np.sqrt(2) / 2 * np.array([-sizes, sizes]).T,
-            points + np.sqrt(2) / 2 * np.array([-sizes, -sizes]).T,
+            points + 0.5 * np.array([sizes, sizes]).T,
+            points + 0.5 * np.array([sizes, -sizes]).T,
+            points + 0.5 * np.array([-sizes, sizes]).T,
+            points + 0.5 * np.array([-sizes, -sizes]).T,
         ],
         axis=0,
     )
     return rect
 
 
-def points_in_box(corners, points, sizes):
+def _points_in_box_3d(
+    box_corners: np.ndarray,
+    points: np.ndarray,
+    sizes: np.ndarray,
+    box_normal: np.ndarray,
+    up_direction: np.ndarray,
+) -> List[int]:
+    """Determine which points are inside of a by a bounding box that is extended
+    along its normal direction.
```

Comment:
This might be easier to understand if described in terms of 2D view of a 3D scene? Something like
```suggestion
    """Determine which points lie inside of a 2D bounding box when viewed from a specific direction.
```

Reply:
I think this is still a bit unclear. Something that should be clarified is that the box is not a parallelepiped, but it has an infinite "side" in the direction of `box_normal`... or if I got it wrong, then it needs to be clarifed otherwise :P

Reply:
Sorry, I hadn't fixed it. The box is actually "infinite" in both directions along the `box_normal`. I will update the docstring now.

Reply:
I've now updated the docstring. Let me know if you think it should be further clarified, @brisvag and @alisterburt !